tests/chainerx_tests/op_utils.py

```diff
@@ -217,8 +304,7 @@ def _make_backend_config(device_name):
     return backend_config
 
 
-def _create_test_entry_function(
-        cls, module, devices, func_suffix, method_name):
+def _create_test_entry_function(cls, module, devices):
```

Comment:
Could you please update the docs for this function, e.g. the removal of `func_suffix`?

Reply:
Fixed.